wiki/magnitude

```diff
@@ -0,0 +1,54 @@
+--- 
+title: Magnitude
+layout: wiki
+---
+
+# Introduction
+
+The Gridcoin network encompasses multiple types of BOINC projects, each one with a different methodology for calculating credit. These differences are reconciled through an innovative mechanism called Magnitude, which subsequently determines reward calculations.
+
+# Methodology
+
+The Gridcoin network has a **Total Magnitude** of 115,000 that is evenly distributed across all whitelisted projects. As there are currently 16 whitelisted projects, this results in a **Project Magnitude** of 7,187.5. This Project Magnitude is distributed among its Gridcoin crunchers in proportion to their RAC. Lastly, each **Cruncher's Magnitude** is calculated by adding the total sum of their magnitudes across all whitelisted projects. This calculation is illustrated through the examples below. 
```

Comment:
The bold on the website is fairly strong, so I'd recommend not bolding these terms
![image](https://user-images.githubusercontent.com/30132912/106334269-4ccbc980-6258-11eb-9922-64be0f17e56b.png)
```suggestion
The Gridcoin network has a Total Magnitude of 115,000 that is evenly distributed across all whitelisted projects. As there are currently 16 whitelisted projects, this results in a Project Magnitude of 7,187.5. This Project Magnitude is distributed among its Gridcoin crunchers in proportion to their RAC. Lastly, each Cruncher's Magnitude is calculated by adding the total sum of their magnitudes across all whitelisted projects. This calculation is illustrated through the examples below. 
```